Rephrase issue tracking and move goals forward (#9)
afrind/moqbof: meeting-materials/bofreq-frindell-media-over-quic.md

# Name: Media Over QUIC (MOQ)
# Description

There are video/media related use-cases that do not appear to be well met by existing protocols. This is a non-WG-forming BOF to discuss these use cases, and determine how to go forward. 

## Issue tracking for this BOF description and agenda

* This BOF description is being maintained in the GitHub repository at https://github.com/afrind/moqbof/.
* Issues to be discussed (and, if possible, resolved) on this BOF description and agenda can be found at https://github.com/afrind/moqbof/issues.
* New issues can be raised in this repository, or on the MOQ mailing list (details below). 

> * The BOF co-chairs (details below) welcome discussion on these issues on the MOQ mailing list (details below). You don't need to wait for the BOF to start, to express an opinion! 

Reviewer: ```suggestion
* The BOF co-chairs (details below) welcome discussion on these issues on the MOQ mailing list (details below). You don't need to wait for the BOF to start to express an opinion! 
```

# BOF Scope

## BoF Goals
The purpose of this BoF is to:

1. Identify media-related use cases with a significant interested community that are not well supported by existing protocols
2. Understand the gaps implementers encounter in using existing protocols to meet the needs of each use case

> 2. Understand whether these existing protocols could reasonably be extended to meet each use case
> 3. If not, is there agreement we should forge a new protocol to meet this use case?

Reviewer: ```suggestion
3. Understand whether these existing protocols could reasonably be extended to meet each use case
4. If not, is there agreement we should forge a new protocol to meet these use cases?
```

### BoF Non-goals

> 1. It is possible that after the BoF we will decide that a single protocol will meet all the needs of the user cases, but we shouldn't work too hard on making that happen, during our discussions. 

Reviewer: ```suggestion
1. It is possible that after the BoF we will decide that a single protocol will meet all the needs of the use cases, but we shouldn't work too hard on making that happen, during our discussions. 
```
